staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2284,6 +2288,25 @@ type SMBiosConfiguration struct {
 	Family       string `json:"family,omitempty"`
 }
 
+type SupportContainerType string
+
+const (
+	// HotplugAttachment is the container resourcs of the hotplug attachment pod
+	HotplugAttachment SupportContainerType = "hotplugAttachment"
+	// ContainerDiskContainer is the container resources used to attach a container disk to the Virtual Machine
+	ContainerDisk SupportContainerType = "containerDisk"
+	// VirtioFS is the container resources used to attach a virtio fs volume to the Virtual Machine
+	VirtioFS SupportContainerType = "virtioFs"
+	// SideCar is the container resources for a SideCar
+	SideCar SupportContainerType = "sideCar"
```

Comment:
I think it is better to have these strings either start with caps or be entirely lowercase.  Slight preference for entirely lowercase